tests/Common/Common.Tests.cs

```diff
@@ -339,6 +339,9 @@ public void TestPropertyChains()
     [Test]
     public void TestStaticClasses()
     {
+        Type staticClassType = typeof(TestStaticClass).GetType();
```

Comment:
.GetType() actually returns a .NET system type here, RuntimeType - that is, the type of the type. You only need typeof(TestStaticClass).